__tests__/Unit/Components/ProgressForm/ProgressHeader.test.tsx

```diff
@@ -0,0 +1,20 @@
+import { render, screen } from '@testing-library/react';
+
+import ProgressHeader from '@/components/ProgressForm/ProgressHeader';
+
+describe('ProgressHeader', () => {
+    test('Should render Text in Progress Header component', () => {
+        render(<ProgressHeader totalMissedUpdates={10} updateType="Standup" />);
+        const missedUpdatesElement = screen.getByText(
+            'Let us try to avoid having zero days'
+        );
+        expect(missedUpdatesElement).toBeInTheDocument();
+    });
+
+    test('Should render Standup Date and time', () => {
+        jest.useFakeTimers().setSystemTime(new Date('17 July 2023'));
+        render(<ProgressHeader totalMissedUpdates={10} updateType="Standup" />);
+        expect(screen.getByTestId('standupTime')).toBeInTheDocument();
+        expect(screen.getByTestId('currentStandupTime')).toBeInTheDocument();
+    });
+});
```

Comment:
Can we please test if the number of missed updates is updated once the standup is submitted - the point 1 addressed from desc of the PR?

Reply:
this can be tested only in standup feature test, here it's not possible to test